test/models/playlist_test.rb

```diff
@@ -0,0 +1,23 @@
+require 'test_helper'
+
+class PlaylistTest < ActiveSupport::TestCase
+  test 'valid if danceanbility beetween 1 and 100' do
+    1.upto(100).each do
+      playlist = Playlist.new(danceability: 10)
+      playlist.valid?
+      assert_empty playlist.errors[:danceability]
+    end
+  end
+
+  test 'invalid if score is less than 1' do
+    playlist = Playlist.new(danceability: 0)
+    playlist.valid?
+    assert_not playlist.errors[:danceability].empty?
+  end
+
+  test 'invalid if score is greater than 100' do
+    playlist = Playlist.new(danceability: 101)
+    playlist.valid?
+    assert_not playlist.errors[:danceability].empty?
```

Comment:
A la lumière des remarques précédentes:

```suggestion
    playlist = Playlist.new(valid_playlist_params.merge(danceability: 101))
    assert_not playlist.valid?
```